lib/brains/forms.py

```diff
@@ -110,3 +128,11 @@ def braintree_data(self):
             self.cleaned_data['bt_signature'],
             self.cleaned_data['bt_payload']
         )
+
+    def clean(self):
+        res = requests.post(settings.BRAINTREE_PROXY + '/parse', self.data)
```

Comment:
Oh, nevermind, I see that you are just passing through the signature and payload